openslides/motions/static/templates/motions/motion-list.html

```diff
@@ -223,7 +223,9 @@ <h1 translate>Motions</h1>
                     ng-options="user.id as user.full_name for user in users"
                     search-contains="true"
                     id="selectSubmitter"
-                    class="form-control">
+                    class="form-control"
+                    data-placeholder-text-multiple="'No submitter selected' | translate"
+                    no-results-text="'No submitter available' | translate">
```

Comment:
Please use consistent strings for no-results-text, e.g. "No submitter/supporter/tag available".